### PR TITLE
Cleaning up main.py

### PR DESCRIPTION
**Pull request by Eludus**
I've made minor changes to better read-ability and simplicity. I also may have screwed up some python syntax as it has been a while so please help me look through to see if everything is working. 

Changes

1. I've added "	`if (not isinstance(message.channel, discord.channel.DMChannel) or message.author == client.user)` " at the start of the message event to just ignore any messages thats sent by the bot OR not in DM. So every if function afterwards does not need to include it. 

2. Same goes for "`if(user == client.user or not isinstance(message.channel, discord.channel.DMChannel)):` " on _reaction_add event.

3. For the on_reaction_add event, the if functions, I made them into elif as only 1 of those condition will ever be met. This will improve code efficiency to not have to loop through every condition

4. In the on_readtion_add event, I added

```
			await msg.add_reaction(one) #add the reactions to the bot so that the user can select a bot
			await msg.add_reaction(two)
			await msg.add_reaction(three)
			await mgs.add_reaction(four)
```

So the bot will add the reaction immediately after the message is sent, instead of adding it when its detected in on_message event. This will remove the need for the janky if function:

```
	# if the channel is a dm, the messsage author isn't the bot itself and the message content matches the same str as below (janky but will be moved to a file soon) 
	elif isinstance(message.channel, discord.channel.DMChannel) and (message.author == client.user) and (message.content == "-----------------\nWhat category would you place your request under?\n\n"+one+" Moderation\n\n"+two+" Tournaments\n\n"+three+" Mentoring\n\n"+four+" Other\n-----------------") and userInputMode != True:
```

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -31,27 +31,23 @@ async def on_message(message):
 	global category
 	global userMessage
 
+	# if the message is not a dm or if message is from bot, return (Do not proceed)
+	if (not isinstance(message.channel, discord.channel.DMChannel) or message.author == client.user):
+		return
+
```
Comment (Eludus) [inline main.py:37]: Ignore any messages thats sent by the bot OR not in DM. So every if function afterwards does not need to include it.

File: main.py [hunk 2]
```diff
@@ -80,30 +76,42 @@ async def clear_react(message):
...
 		global category
-		# when a message has the "tick" emoji added, the user id is not the bot's and the channel is a dm 
-		if reaction.emoji == tick and user != client.user and isinstance(message.channel, discord.channel.DMChannel):
-			await reaction.message.channel.send("-----------------\nWhat category would you place your request under?\n\n"+one+" Moderation\n\n"+two+" Tournaments\n\n"+three+" Mentoring\n\n"+four+" Other\n-----------------")
-
-		# when a message has the "one" emoji added, the user id is not the bot's and the channel is a dm
-		if reaction.emoji == one and user != client.user and isinstance(message.channel, discord.channel.DMChannel):
+		
+		#  If the user id is the bot's or if the channel is not a dm, return
+		if(user == client.user or not isinstance(message.channel, discord.channel.DMChannel)):
```
Comment (Eludus) [inline main.py:81]: Ignore any messages thats sent by the bot OR not in DM. So every if function afterwards does not need to include it.

File: main.py [hunk 3]
```diff
@@ -31,27 +31,23 @@ async def on_message(message):
...
 		await msg.add_reaction(tick)
 
-	# if the channel is a dm, the messsage author isn't the bot itself and the message content matches the same str as below (janky but will be moved to a file soon) 
-	elif isinstance(message.channel, discord.channel.DMChannel) and (message.author == 
-		client.user) and (message.content == "-----------------\nWhat category would you place your request under?\n\n"+one+" Moderation\n\n"+two+" Tournaments\n\n"+three+" Mentoring\n\n"+four+" Other\n-----------------") and userInputMode != True:
-		await message.add_reaction(one) #add the reactions to the bot so that the user can select a bot
-		await message.add_reaction(two)
-		await message.add_reaction(three)
-		await message.add_reaction(four)
-
```
Comment (Eludus) [inline main.py:52]: Removed,
Made the bot add comment ight after the message was sent in on_add_reaction event

**Comment by IsaacWP121** [comment]
thanks for the work, I overlooked a lot of those things

**Pull request merged**